xla/hlo/transforms/simplifiers/algebraic_simplifier_test.cc

```diff
@@ -12688,5 +12688,31 @@ TEST_F(AlgebraicSimplifierTest, TestNew123) {
   EXPECT_FALSE(simplifier.Run(module.get()).value());
 }
 
+TEST_F(AlgebraicSimplifierTest,
+       ReducePrecisionWithSameDatatypeShouldBeRemoved) {
```

Comment:
`ReducePrecisionWithSamePrecisionAsDataTypeShouldBeRemoved` (the name is not precise enough, reduce precision is always on the "same" data type for i/o :)